ledger/participant-integration-api/src/test/suite/scala/platform/apiserver/GrpcServerSpec.scala

```diff
@@ -11,16 +11,18 @@ import com.daml.ledger.client.configuration.{
   LedgerClientConfiguration,
   LedgerIdRequirement
 }
+import com.daml.ledger.resources.{ResourceContext, ResourceOwner}
 import com.daml.metrics.Metrics
 import com.daml.platform.apiserver.GrpcServerSpec._
 import com.daml.platform.hello.{HelloRequest, HelloServiceGrpc}
 import com.daml.ports.Port
-import com.daml.resources.ResourceOwner
 import com.google.protobuf.ByteString
 import io.grpc.ManagedChannel
 import org.scalatest.{AsyncWordSpec, Matchers}
 
 final class GrpcServerSpec extends AsyncWordSpec with Matchers {
+  private implicit val resourceContext: ResourceContext = ResourceContext(executionContext)
```

Comment:
Should we have our own `AsyncWordSpec` that uses the `ResourceContext`? Having two implicit contexts in the mix seems like a recipe for trouble.

Reply:
I could add a helper trait: `with SpecResourceContext` or something.